package.json

```diff
@@ -11,8 +11,8 @@
     "compile:css": "yarn clean-dist && cleancss -o dist/compressed.css node_modules/material-components-web/dist/material-components-web.css node_modules/dialog-polyfill/dialog-polyfill.css src/libs/css/app.css",
     "dev": "yarn compile:dev && yarn start:dev",
     "lint": "eslint src/libs --ext=js --ext=jsx && eslint src/demos --ext=js --ext=jsx",
-    "check": "yarn run lint && yarn run jest",
-    "test": "yarn run jest",
+    "check": "yarn lint && yarn jest",
```

Comment:
I insist, it should be `yarn test` not `yarn jest`

Reply:
Sorry haven't see it ;)